demo/src/app/components/typeahead/demos/http/typeahead-http.ts

```diff
@@ -41,6 +42,7 @@ export class NgbdTypeaheadHttp {
   model: any;
   searching = false;
   searchFailed = false;
+  hideSearchingWhenUnsubsctibed = new Observable(() => () => this.searching = false);
```

Comment:
I'm not sure if it's possible to better handle unsubscription of `search` observable